strongbox-user-management/pom.xml

```diff
@@ -199,12 +208,23 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>2.0.2-beta</version>
```

Comment:
Could you please extract the version to the `strongbox-parent`? We don't use versions in the `pom.xml` files, as we inherit them from there.